modules/modal/react/stories/stories_VisualTesting.tsx

```diff
@@ -17,12 +18,33 @@ export default withSnapshotsEnabled({
 
 const noop = () => {}; // eslint-disable-line no-empty-function
 
+const TestContent = () => {
+  const content = (
+    <div
+      style={{
+        color: 'white',
+        zIndex: 1,
+        position: 'relative',
+      }}
```

Comment:
This ensures the Modal always renders on top of other content. This proves it still works. It would show as a visual regression otherwise